pkg/rpc/client/wsclient.go

```diff
@@ -24,7 +24,7 @@ import (
 // connection to the server and at the same time is exposes some functionality
 // that is only provided via websockets (like event subscription mechanism).
 type WSClient struct {
-	Client
+	*Client
```

Comment:
I'd rather keep it this way, you can create an internal `new(*Client, ctx...)` and pass a pointer there allocating outside of it.

Reply:
This way we still need to dereference `*Client` after that which leads to the same warning. Could you see the updated version?

Reply:
Fixed.